pointsource_imaging/NIRCam_Basic_PSF_Photometry_solution.ipynb

```diff
@@ -74,7 +74,7 @@
    "id": "95891849",
```

Comment:
<p>The plot text functions you call in <code>plot_epsf()</code> don't have <code>font</code> as an argument. Instead, try <code>fontsize=NUMBER</code> where you have <code>font={'size': NUMBER}</code>.<span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='115'/>